### DISCUSSION TITLE
Moving subsets from Covidence to ASreview

### DISCUSSION THREAD

**Discussion by jenshk77** [asreview/asreview#1936] [Q&A] [answered]
I am about to start screening more than 10.000 titles & abstracts in Covidence - and I would like to just screen ('train') a smaller subset (~1.500) and then move them all to ASreview and run a simulation on the rest.
Do any of you have experience with this - and are there any do's and don'ts?

**Comment by Rensvandeschoot** [comment, marked as answer]
If you import a [partly labeled dataset](https://asreview.readthedocs.io/en/latest/data_labeled.html) into ASReview, it will automatically recognize the labels and will use these to train the first iteration of the model.

If you have subsets of data, you might be interested in the [Compose](https://github.com/asreview/asreview-datatools?tab=readme-ov-file#data-compose-experimental) functionality to combine different datasets into one big file, which you can import into ASReview. 

I am not a user of Covidence, so for that part I cannot help you :-)

**Reply by jenshk77** [reply]
Thank you for answering quickly! :-)
